staging/src/k8s.io/apiextensions-apiserver/pkg/controller/finalizer/crd_finalizer.go

```diff
@@ -79,7 +79,7 @@ func NewCRDFinalizer(
 		crdLister:      crdInformer.Lister(),
 		crdSynced:      crdInformer.Informer().HasSynced,
 		crClientGetter: crClientGetter,
-		queue:          workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "CustomResourceDefinition-CRDFinalizer"),
+		queue:          workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "crd_finalizer"),
```

Comment:
crd_finalizer_controller?

Reply:
crd_finalizer_controller sounds like a controller for crd finalizer to me

also we tend to omit "controller" from the name for other in-tree controllers: https://github.com/kubernetes/kubernetes/search?q=NewNamedRateLimitingQueue&unscoped_q=NewNamedRateLimitingQueue